OpenTabletDriver.Web/Views/Wiki/FAQ/Windows.cshtml

```diff
@@ -77,4 +77,16 @@
         If this does not solve the issue, check that you are not using the "Administrator" named account as it overrides User Account
         Control.
     </p>
+</div>
+
+<hr/>
+
+<h3 class="wiki-nav-item pb-2" id="rtss">
+    Why do i get a blank screen when opening OpenTabletDriver?
+</h3>
+<div class="ms-3">
+    <p>
+        This can be caused by RivaTuner Statistics Server attempting to hook onto OpenTabletDriver, preventing the UX
+        from responding due to failed hooks. If you use this application make sure it doesn't hook onto OpenTabletDriver.
```

Comment:
```suggestion
        from responding due to failed hooks. If you use this application, make sure it doesn't hook onto OpenTabletDriver.
```

Reply:
Describe how to not hook onto OpenTabletDriver, or refer users to links on how to do so.